wad.md

```diff
@@ -82,14 +82,15 @@ WAD versions 2 and 3 include all fields.
 |  12 |    4 | u32    | compressed size                        |
 |  16 |    4 | u32    | uncompressed size                      |
 |  20 |    1 | u8     | compression type                       |
-|  21 |    1 | bool   | *unknown*                              |
-|  22 |    2 | u16    | *unknown*                              |
+|  21 |    1 | bool   | [is duplicated ?](#entry-duplication)  |
```

Comment:
`is duplicated ?` -> `set for duplicated entries`
(I prefer staying with affirmative tense for fields description.)

Reply:
I don't think I quite understand what you mean.

Reply:
I just sugges to remove the question, for homogeneity with other descriptions.

Reply:
done